### PR TITLE
Determine libs dir git SHA automatically instead of hardcoding it

### PR DESCRIPTION
**Pull request by thomcc**
This was out of date. @ncalexan, this is what you meant by figuring it out automatically right?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -80,13 +80,8 @@ task clean(type: Delete) {
...
 import com.sun.jna.Platform as DefaultPlatform
 
-// This is the output `git rev-parse HEAD:libs`.  We could discover this automatically, but
-// let's keep things simple and bump this manually for a while.  If and when we grow SNAPSHOT
-// support on maven.mozilla.org, we can invest in a better approach for versioning these
-// prerequisite libraries.
-//
 // Set this to `= null` in order to use libs from the source directory.
-ext.libsGitSha = '5963858cbc4a55b295dd69efd97bc043259241c5'
+ext.libsGitSha = "git rev-parse HEAD:libs".execute().text.trim()
```
Comment (ncalexan) [inline build.gradle]: You're not substituting text, so use single quotes here.

**Comment by thomcc** [comment]
Good point. How about this?

**Comment by thomcc** [comment]
> So my belief is that what you had before should work.

It does not if you have committed changes in `libs`. I tried modifying a file in `libs`, committinng, and then doing `./gradle build`, and it failed (because the SHA of `libs` dir did not exist remotely)

**Comment by ncalexan** [comment]
> > So my belief is that what you had before should work.
> 
> It does not if you have committed changes in `libs`. I tried modifying a file in `libs`, committing, and then doing `./gradle build`, and it failed (because the SHA of `libs` dir did not exist remotely)

That makes sense.  Roll on with your updated patch, although I'd still prefer the "CI" environment guard for clarity (or a good comment, if you prefer).

**Comment by thomcc** [comment]
That's not a CI specific change. Actually, CI handles this separately, so it's not even relevant for CI. I guess I had actually misunderstood your comment initially, but realized a bug in that misunderstanding.

**Pull request merged**